src/ray/gcs/gcs_client/test/global_state_accessor_test.cc

```diff
@@ -272,6 +272,10 @@ TEST_F(GlobalStateAccessorTest, TestWorkerTable) {
 }  // namespace ray
 
 int main(int argc, char **argv) {
+  InitShutdownRAII ray_log_shutdown_raii(ray::RayLog::StartRayLog,
```

Comment:
Why do we need this?

Reply:
It's not necessary, but it helps to print out the stack trace when the test crashes.